### DISCUSSION TITLE
Version control on the guides themselves?

### DISCUSSION THREAD

**Discussion by tdhc153** [tdhc153/starmast#2] [Ideas]
Do we think we should have version control on all the guides? This could be part of the summary at the start, or at the end in a box containing our contact details.

**Comment by Joseph-Edwards** [comment]
Could clarify what you mean by this please? Aren't the guides already being version controlled by virtue of being hosted on GitHub?

**Reply by tdhc153** [reply]
I really mean version control/a change log on the published output, which is visible to the reader. For instance, should there be a box at the end saying something like (v1 initial publish 03/07, v1.1 typos corrected 10/07), so that the reader can compare any printed/saved version to the newest version? These are present on (for instance) University policy documents: see https://www.st-andrews.ac.uk/policy/academic-policies-assessment-examination-and-award-ug-senate-regulations/2022-2023/ug-senate-regs.pdf for an example.

I am in favour, as it would underline the fact that this would be an ongoing concern.

**Reply by Joseph-Edwards** [reply]
I see. Thanks for explaining that. I'm also in favour; I think it's important for the reader to know that these documents can be edited and that they aren't necessarily in their final form.

My only concern is that, if this becomes too large, it might detract from the guide itself. Could there be a file hosted on GitHub that contains full details of all changes to the output files, and then on each output file the box at the bottom contains the most recent changes? Then there could be a link that directs the reader to the big file for full details?
